gio/Gir.toml

```diff
@@ -408,6 +408,11 @@ generate_builder = true
     #return vec of u8
     manual = true
 
+    [[object.function]]
+    name = "read_line_finish_utf8"
+    #finish functions not needed
+    manual = true
```

Comment:
This needs to be done in https://github.com/gtk-rs/gir/ directly and automatically

Reply:
Do you have pointers on where to start? the gir program is not small :)

Reply:
I'd search for `_async` and `_finish` as strings. That should lead you to the code where the gio-style async functions are handled.

Somewhere in `src/analysis` you should be able to mark the `finish()` functions as ignored then.

Reply:
I left this modification to the `Gir.toml` for now in this new commit to check if it fixes the build, but I did a parallel PR to gir: https://github.com/gtk-rs/gir/pull/1031

I will remove this once the gir sub-repo is updated